### PR TITLE
Unix status codes

### PR DESCRIPTION
**Pull request by DoubleCouponDay**
@Bobscorn please review changes and approve if the changes are satisfactory.

### CODE REVIEW

**Comment by colethedj** [comment]
LGTM

File: Rust_part/src/secrettoken.rs [hunk 1]
```diff
@@ -7,3 +7,7 @@ pub fn gettoken() -> String {
 pub fn getwatchertoken() -> String {
     var("TRAMPOLINE").unwrap()
 }
+
+pub fn getchannelid() -> u64 {
+    787470274261549056
```
Comment (Bobscorn) [inline Rust_part/src/secrettoken.rs:12]: Hard coded channel id?

File: C_part/src/nsvg/copy.c [hunk 2]
```diff
@@ -2,7 +2,7 @@
 #include <nanosvg.h>
 #include <stdio.h>
 #include <stdlib.h>
-#include <unistd.h>
+//#include <unistd.h> //linux only
```
[the remark this answers is not in the document]
Comment (DoubleCouponDay) [reply C_part/src/nsvg/copy.c:5]: this line should be removed

File: C_part/src/entrypoint.c [hunk 3]
```diff
@@ -95,24 +96,27 @@ int entrypoint(int argc, char* argv[]) {
 		threshold = (float)atof(argv[4]);	
 		
 	if (threshold < 0.f)
-		threshold = DEFAULT_THRESHOLD;
+		threshold = 1;
```
[the remark this answers is not in the document]
Comment (DoubleCouponDay) [reply C_part/src/entrypoint.c:99]: Yes this should have been DEFAULT_THRESHOLD.

File: Rust_part/src/secrettoken.rs (hunk 1 above)
Comment (DoubleCouponDay) [reply Rust_part/src/secrettoken.rs:12]: I'll try to fix in #18.

**Pull request merged**